tests/h/search/index_test.py

```diff
@@ -93,6 +106,141 @@ def test_you_can_filter_annotations_by_creation_time(self, factories, index, sea
         response = search.filter("range", created={"gte": before}).execute()
         assert SearchResponseWithIDs([annotation.id]) == response
 
+    def test_you_can_filter_annotations_by_updated_time(self, factories, index, search):
+        update_time = datetime.datetime.now()
+        annotation = factories.Annotation.build(id="test_annotation_id", updated=update_time)
+
+        index(annotation)
+
+        response = search.filter("range", updated={"gte": update_time}).execute()
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_id(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_ann_id")
+
+        index(annotation)
+
+        response = search.filter("term", id="test_ann_id").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    @pytest.mark.parametrize('user_search_str', [
+        'acct:someone@example.com',
+        'someone',
+        'someone@example.com'
+    ])
+    def test_you_can_filter_annotations_by_user(self, factories, index, search, user_search_str):
+        annotation = factories.Annotation.build(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        response = search.filter("term", user=user_search_str).execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_make_aggregations_on_user_raw(self, factories, index, search):
+        annotation_1 = factories.Annotation.build(userid="acct:someone@example.com")
+        annotation_2 = factories.Annotation.build(userid="acct:Someone@example.com")
+
+        index(annotation_1)
+        index(annotation_2)
+
+        a = elasticsearch1_dsl.A('terms', field='user_raw')
+        search.aggs.bucket('user_raw_terms', a)
+
+        response = search.execute()
+
+        user_bucket_1 = next(bucket for bucket in response.aggregations.user_raw_terms.buckets if bucket["key"] == "acct:someone@example.com")
+        user_bucket_2 = next(bucket for bucket in response.aggregations.user_raw_terms.buckets if bucket["key"] == "acct:Someone@example.com")
```

Comment:
Please try to keep lines < 100 chars at most, as they are fiddly to read in GitHub's diff viewer otherwise.